src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ReturnItemsCard.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Card,
+  CardContent,
+  Checkbox,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TextField
+} from "@material-ui/core";
+import Money from "@saleor/components/Money";
+import Skeleton from "@saleor/components/Skeleton";
+import TableCellAvatar from "@saleor/components/TableCellAvatar";
+import { FormsetChange } from "@saleor/hooks/useFormset";
+import { renderCollection } from "@saleor/misc";
+import {
+  OrderDetails_order,
+  OrderDetails_order_lines
+} from "@saleor/orders/types/OrderDetails";
+import React, { CSSProperties } from "react";
+import { defineMessages, FormattedMessage, useIntl } from "react-intl";
+
+import { FormsetQuantityData, FormsetReplacementData } from "../form";
+import { getById } from "../utils";
+import CardTitle from "./CardTitle";
+import MaximalButton from "./MaximalButton";
+
+const useStyles = makeStyles(
+  theme => {
+    const inputPadding: CSSProperties = {
+      paddingBottom: theme.spacing(2),
+      paddingTop: theme.spacing(2)
+    };
+
+    return {
+      cartContent: {
+        paddingBottom: 0,
+        paddingTop: 0
+      },
+      notice: {
+        marginBottom: theme.spacing(1),
+        marginTop: theme.spacing(2)
+      },
+      quantityInnerInput: {
+        ...inputPadding
+      },
+      quantityInnerInputNoRemaining: {
+        paddingRight: 0
+      },
+      remainingQuantity: {
+        ...inputPadding,
+        color: theme.palette.text.secondary,
+        whiteSpace: "nowrap"
+      },
+      setMaximalQuantityButton: {
+        marginBottom: theme.spacing(1),
+        marginTop: theme.spacing(2),
+        padding: 0
+      }
+    };
+  },
+  { name: "ItemsCard" }
+);
+
+const messages = defineMessages({
```

Comment:
Those labels seems to be duplicated in other components. How about extracting the to one, common fil?

Reply:
Transifex will merge to one string if context is the same

Reply:
-> big refactor